src/routes/(site)/about/officer-profile.svelte

```diff
@@ -161,6 +204,17 @@
                   </a>
                 </p>
               {/if}
+              {#if officerSocials.youtube}
+                <p class="officer-youtube">
+                  <a
+                    target="blank"
+                    title={officerSocials.youtube}
+                    href="https://www.youtube.com/@{officerSocials.youtube}"
+                  >
+                    <BwIcon src="/assets/svg/youtube.svg" alt="Youtube icon" /><span>Youtube</span>
```

Comment:
Nit: YouTube has a capital T.
```suggestion
                    <BwIcon src="/assets/svg/youtube.svg" alt="YouTube icon" /><span>YouTube</span>
```